packages/vitest/src/integrations/chai/jest-expect.ts

```diff
@@ -1,5 +1,5 @@
 import c from 'picocolors'
-import { AssertionError } from 'chai'
+import { AssertionError, util as chaiUtil } from 'chai'
```

Comment:
We already have utils on 

https://github.com/vitest-dev/vitest/blob/a43df9e49f2fe98bb2791d12d1c9ac3350f1ac01/packages/vitest/src/integrations/chai/jest-expect.ts#L35

Reply:
Sorry too enthusiastic, I don't look enough at what's already there... I made the change.